packages/@react-stately/slider/src/useSliderState.ts

```diff
@@ -17,52 +17,125 @@ import {useNumberFormatter} from '@react-aria/i18n';
+   * @param index
+   * @param dragging
+   */
+  setThumbDragging(index: number, dragging: boolean): void,
+
+  /**
+   * Currently-focused thumb index.
+   */
   readonly focusedThumb: number | undefined,
-  setFocusedThumb: (index: number | undefined) => void,
-
-  // Returns the value offset as a percentage from 0 to 1.
-  getThumbPercent: (index: number) => number,
-  getValuePercent: (value: number) => number,
-
-  // Returns the string label for the value, per props.formatOptions
-  getThumbValueLabel: (index: number) => string,
-  getFormattedValue: (value: number) => string,
-
-  // Returns the min and max values for the index
-  getThumbMinValue: (index: number) => number,
-  getThumbMaxValue: (index: number) => number,
-
-  // Converts a percent along track (between 0 and 1) to the corresponding value
-  getPercentValue: (percent: number) => number,
-
-  // editable
-  isThumbEditable: (index: number) => boolean,
-  setThumbEditable: (index: number, editable: boolean) => void,
-
-  // The step amount for the slider
+  /**
+   * Set focused true on specified thumb. This will remove focus from
+   * any thumb that had it before.
+   * @param index
+   */
+  setFocusedThumb(index: number | undefined): void,
+
+  /**
+   * Returns the specified thumb's value as a percentage from 0 to 1.
+   * @param index
+   */
+  getThumbPercent(index: number): number,
+
+  /**
+   * Returns the value as a percent between the min and max of the slider.
+   * @param index
+   */
+  getValuePercent(value: number): number,
+
+  /**
+   * Returns the string label for the specified thumb's value, per props.formatOptions.
+   * @param index
+   */
+  getThumbValueLabel(index: number): string,
+
+  /**
+   * Returns the string label for the value, per props.formatOptions.
+   * @param index
+   */
+  getFormattedValue(value: number): string,
+
+  /**
+   * Returns the min allowed value for the specified thumb.
+   * @param index
+   */
+  getThumbMinValue(index: number): number,
+
+  /**
+   * Returns the max allowed value for the specified thumb.
+   * @param index
+   */
+  getThumbMaxValue(index: number): number,
+
+  /**
+   * Converts a percent along track (between 0 and 1) to the corresponding value.
+   * @param percent
+   */
+  getPercentValue(percent: number): number,
+
+  /**
+   * Returns if the specified thumb is editable.
+   * @param index
+   */
+  isThumbEditable(index: number): boolean,
+
+  /**
+   * Set the specified thumb's editable state.
+   * @param index
+   * @param editable
+   */
+  setThumbEditable(index: number, editable: boolean): void,
+
+  /**
+   * The step amount for the slider.
+   */
   readonly step: number
 }
 
 export const DEFAULT_MIN_VALUE = 0;
 export const DEFAULT_MAX_VALUE = 100;
 export const DEFAULT_STEP_VALUE = 1;
 
+/**
+ * Provides state management for a slider component. Stores values for all thumbs
+ * and formats values for localization. Also provides methods to update the position
```

Comment:
`State management methods to update...` or "helper"? Trying to think of a way not to say "also".